pyproject.toml

```
@@ -19,6 +19,12 @@ sources = ["src/testbed"]
 test_sources = ["tests"]
 
 requires = [
+    # Android can't process .pth files (see chaquo/chaquopy#1338). This also
+    # means we can only test Linux on Python3.13+, because Android reports as
+    # "Linux" on Python3.12 and earlier.
+    """x-pth-tester; \
+        (platform_system != 'Linux' and platform_system != 'Android') \
+        or (platform_system == 'Linux' and python_version >= '3.13')""",
```

The Chaquopy bug is about .pth files earlier in the sequence (app, requirements, stdlib) accessing modules from later in the sequence. That doesn't occur in this test.

```suggestion
    "x-pth-tester",

```